great-idea/index.html

```diff
@@ -15,9 +15,63 @@
 </head>
 
   <body>
-
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
+      <div class="container">
+      <header>
+       <nav>
+        <a href="#">Services</a>
+        <a href="#">Product</a>
+        <a href="#">vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+       </nav>
```

Comment:
what i like to do here is have the CTA as part of the main body and then my header looks like this
<header>
   <nav>
      <a>
   </nav>
   <img>
</header>

Then you can just slap the flex on your header with space between  and yoru nav is on one side and the logo on the other.

Reply:
Thanks! I will definitely be practicing with that.